mmocr/models/textdet/losses/fce_loss.py

```diff
@@ -68,25 +68,23 @@ def forward(self, preds, _, p3_maps, p4_maps, p5_maps):
     def forward_single(self, pred, gt):
         cls_pred, reg_pred = pred[0], pred[1]
 
+        k = 2 * self.fourier_degree + 1
         tr_pred = cls_pred[:, :2, :, :].permute(0, 2, 3, 1)\
```

Comment:
Can the `permute` and `contiguous` done beforehand, as I commented in the earlier pull request?

Reply:
ok